reference/blob/models/device-logs-metadatalist.v1.yaml

```diff
@@ -0,0 +1,6 @@
+title: Device logs json list
+type: array
+readOnly: true
+minItems: 0
```

Comment:
Is there a `maxItems`?

Reply:
hmmm - as this is returned data is that as important? i.e. if we set arbitrary max then we would need to stick to that in the code. Thoughts?

fyi we have the same issue for `reference/blob/models/metadatalist.v1.yaml`